repos/system_upgrade/el7toel8/libraries/tests/test_tcpwrappersutils.py

```diff
@@ -92,7 +92,7 @@ def test_config_applies_to_daemon_with_host_except():
     assert lib.config_applies_to_daemon(facts, 'vsftpd') is True
 
     daemon_list = DaemonList(value=['all', 'except', 'all', 'except', 'all',
-                                    'except' 'vsftpd@localhost'])
+                                    'except', 'vsftpd@localhost'])
```

Comment:
Nice catch! So we had the trouble with the test all that time?

Reply:
We do not having trouble with unit test, because the old code -force- a implicit string concatenation, so we continue having list of strings but lost the last 2 values..